app/Repositories/Eloquents/NotificationEloquentRepository.php

```diff
@@ -53,7 +53,8 @@ public function getNotifications($with = [], $data = [], $dataSelect = ['*'])
                         if ($book) {
                             $message = config('view.notifications.' . $record->target->type) . $book->title;
                             $record = array_add($record, 'message', $message);
-                            if ($record->target->type == config('view.request.waiting')) {
+                            if ($record->target->type == config('view.request.waiting') || 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/NotificationEloquentRepository.php</br>**Line**: 56</br>Whitespace found at end of line